packages/react-conformance/src/defaultErrorMessages.tsx

```diff
@@ -488,6 +488,23 @@ export const defaultErrorMessages = {
     });
   },
 
+  'consistent-callback-args': (testInfo: IsConformantOptions, invalidProps: string[]) => {
+    const { displayName } = testInfo;
+    const { testErrorInfo, resolveInfo } = errorMessageColors;
+
+    return getErrorMessage({
+      displayName,
+      overview: 'uses non-standard callback arguments.',
+      details: ['These callback(s) need have two params:', testErrorInfo(formatArray(invalidProps))],
+      suggestions: [
+        `Ensure that callbacks ${resolveInfo(displayName + `'s`)} have two params (an event and data object)`,
+        `If the name is correct, add the prop to isConformant ${resolveInfo(
```

Comment:
I think this would help with @ling1726's question about what "name" is?
```suggestion
        `Ensure that ${resolveInfo(displayName + `'s`)} callbacks have two params (an event and data object)`,
        `If this prop is intended to have a different signature, add the prop to isConformant ${resolveInfo(
```

Reply:
Yeah, much better, thanks 👍